components/webgpu/wgpu_thread.rs

```diff
@@ -218,7 +223,9 @@ impl WGPU {
                             Ok(())
                         };
 
+                        // invalidate command buffer too
                         self.encoder_record_error(command_encoder_id, &result);
```

Comment:
what do you mean? Also a nit for another time: that method should also be called `maybe_`.

Reply:
Specs says that we need to invalidate command encoder on error, but wgpu does not do this (yet). So we consider command encoder invalid if there is error stored in hashmap of error encoders. When command encoder is finished into command buffer (wgpu uses same id via id transmutes) we need to make sure that invalid command encoder also finishes into invalid command buffer (or if there is error when doing finish) so we reuse same error hashmap for this.